### DISCUSSION TITLE
Frigate 13 beta 7 - Changing TensorRT Models

### DISCUSSION THREAD

**Discussion by twister36** [blakeblackshear/frigate#9067] [Ask A Question]
I'm looking to change my TensorRT model from the Default of yolov7-320 due to too many false positives. I do see that we have list of the available models in the Beta documentation, but do we have a matrix that scores each model based off speed vs accuracy?

https://deploy-preview-6262--frigate-docs.netlify.app/configuration/object_detectors#generate-models

**Comment by NickM-27** [comment]
Accuracy from these models is not necessarily consistent from camera to camera, different users have different experiences

**Comment by twister36** [comment]
> Accuracy from these models is not necessarily consistent from camera to camera, different users have different experiences

Well the following website appears to describe the difference in some of the models:
https://viso.ai/deep-learning/yolov7-guide/

and here's information on YOLOv7:
https://docs.ultralytics.com/models/yolov7/

**Comment by Sceptersax** [comment]
@twister36, what did you end up using?

**Reply by twister36** [reply]
I actually fell back to using my Google Coral... Even though it's slower than my Nvidia GPU, it's more accurate on properly identifying my objects.
